xc7/utils/prjxray_create_synth_tiles.py

```diff
@@ -106,44 +151,42 @@ def main():
             }
         )
 
-    # Find two VBRK's in the corner of the fabric to use as the synthetic VCC/
-    # GND source.
-    vbrk_loc = None
-    vbrk_tile = None
-    vbrk2_loc = None
-    vbrk2_tile = None
-    for tile in g.tiles():
-        if tile in vbrk_in_use:
-            continue
+    # Find two VBRK's in the corner of the fabric to use as the synthetic VCC
+    loc_min = GridLoc(g.dims()[0], g.dims()[2])
+    loc_max = GridLoc(g.dims()[1], g.dims()[3])
```

Comment:
Keep them to the left side of the grid, e.g. use min x dim for both.

Reply:
Done.